pkg/kubelet/dockertools/manager.go

```diff
@@ -560,7 +569,10 @@ func (dm *DockerManager) runContainer(
 	}
 	namespacedName := types.NamespacedName{pod.Namespace, pod.Name}
 	labels := map[string]string{
-		"io.kubernetes.pod.name": namespacedName.String(),
+		kubernetesNameLabel: namespacedName.String(),
+	}
+	if pod.Spec.TerminationGracePeriodSeconds != nil {
+		labels[kubernetesTerminationGracePeriodLabel] = strconv.FormatInt(*pod.Spec.TerminationGracePeriodSeconds, 10)
```

Comment:
What is the reason to store the termination grace period as a docker label?

What will happen if the user updates the termination grace period?